book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- Kindly make sure a method for adding rentals is present. 
Hint: ```
`def add_rental(person, date)`
```
- Please make sure the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
Hint: ```
Rental.new(date, self, person)
```